docs/changes/units/17929.api.rst

```diff
@@ -1,3 +1,3 @@
-Accessing the contents of the ``deprecated`` module now emits deprecation
+Accessing the contents of the ``units.deprecated`` module now emits deprecation
```

Comment:
I thought that the fact that this entry is in the `astropy.units` section would make it clear enough that the `deprecated` module should be understood to be relative to `astropy.units`, i.e. it is actually `astropy.units.deprecated`.

Reply:
My bad that I know that there is also `utils.deprecated` and while cat-ting out the snippets it wasn't obvious what this was about.
If you want to _actually_ nitpick `astropy.units.deprecated` is indeed the most appropriate, but dropping astropy would result in an unambiguous entry already therefore I suggested that one.